SeaPublicWebsite/Views/StaticPages/PrivacyPolicy.cshtml

```diff
@@ -0,0 +1,184 @@
+@using SeaPublicWebsite.Controllers
+@{
+    ViewBag.Title = $"Privacy Notice on {Constants.SERVICE_NAME}";
+}
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <h1 class="govuk-heading-xl">
+            Privacy Notice
+        </h1>
+        <p class="govuk-body">
+            This notice sets out how we will use your personal data, and your rights. It is made under Articles 13
+            and/or 14 of the UK General Data Protection Regulation (UK GDPR).
+        </p>
+        <h2 class="govuk-heading-m">
+            YOUR DATA
+        </h2>
+        <p class="govuk-body">
+            This service does not automatically collect or store any personal data about you.
+        </p>
+        <p class="govuk-body">
+            We may process the following personal data:
+        </p>
+        <p class="govuk-body">
+            Your name, email address and any other information sent directly to us via our email feedback function.
+        </p>
+        <p class="govuk-body">
+            We may also collect and process anonymised information as outlined in our
+            <a class="govuk-link" href="@Url.Action(nameof(CookieController.CookieSettings_Get), "Cookie")">
+                cookie policy
+            </a>.
+        </p>
+
+        <h2 class="govuk-heading-s">
```

Comment:
If this is smaller than the 'your data' header, I believe it should be a h3 (and everywhere below). Katherine might know better.

Reply:
That makes sense, I think we can overrule the heatpump page here